Fundamentals/Backoffice/Property-Editors/Built-in-Property-Editors/Block-Grid-Editor/index.md

```diff
@@ -0,0 +1,670 @@
+
+## Configure the Block Grid
+
+The Block Grid property editor is configured in the same way standard property editors, which is via the Data Types backoffice interface.
+
+To set up your Block Grid Editor property, follow these steps:
+
+1. Navigate to the **Settings** section in the Umbraco backoffice.
+2. Locate and right-click the **Data Types** folder.
+3. **Create a new Data Type**.
+4. Select **Block Grid** from the list of available property editors.
+
+You will see the configuration options for a Block Grid as shown below.
+
+![Block Grid - Data Type Definition](images/BlockListEditor_DataType.jpg)
+> TODO: screenshot
+
+The Data Type editor allows you to configure the following properties:
+
+- **Blocks** - Define the Block Types to be available for use in the property. Read more on how to set up Block Types below.
+- **Amount** - Sets the minimum and/or the maximum number of blocks that should be allowed at the root of the layout.
+- **Live editing mode** - Enabling this will enable you to see the changes you make as you are making them.
+- **Editor width** - Overwrite the width of the property editor. This field takes any valid CSS value for "max-width".
+- **Grid Columns** - Define the number of columns in your grid layout. The default is 12 columns.
+- **Layout Stylesheet** - Replace the built-in Layout Stylesheet. Additionally, you can retrieve the default layout stylesheet to use as a base for your own or inspiration for writing your own.
+- **Create Button Label** - Overwrite the label of the create button.
+
+## Setup Block Types
+
+Block Types are based on **Element Types**. These can be created beforehand or while setting up your Block Types.
+
+Once you have added an Element Type as a Block Type on your Block Grid Data Type you have the option to configure it.
+
+![Block List - Data Type Block Configuration](images/BlockListEditor_DataType_Blocks.png)
+> TODO: screenshot
+
+### Groups
+Blocks can also be grouped, this will be visible in the Block Catalogue. This can also be used to allow a group of Blocks in a Area.
+
+## Block Configuration
+Each Block has a set of properties that are optional to configure. These are described below.
+
+### General
+
+Customize the user experience for your content editors when they work with the blocks in the Content section.
+
+- **Label** - Define a label for the appearance of the Block in the editor. The label can use AngularJS template-string-syntax to display values of properties.
+  :::tip
+  Label example: "My Block {{myPropertyAlias}}" will be shown as: "My Block FooBar". 
+  
+  You can also use more advanced expression using AngularJS filters, like `{{myPropertyAlias | limitTo:100}}` or for a property using Richtext editor `{{myPropertyAlias | ncRichText | truncate:true:100}}`. It is also possible to use properties from the Settings model by using `{{$settings.propertyAlias}}`.
+  :::
```

Comment:
Perhaps link to more documentation or the community blog post from Joe @nielslyngsoe & @sofietoft 
https://joe.gl/ombek/blog/umbraco-angularjs-filter-cheat-sheet/

Reply:
Done 👍